### DISCUSSION TITLE
Jobs at Green Software Foundation

### DISCUSSION THREAD

**Discussion by adamj89** [Green-Software-Foundation/discussions#24] [Announcements]
***EDIT:*** Both positions have been filled!
We'll share any future opportunities at GSF on this Announcement discussion and on our LinkedIn.

We are hiring at the Green Software Foundation! 

Check out our roles for ~[Senior Content PM](https://grnsft.org/jobs/contentPM)~ and ~[Senior Technical PM](https://grnsft.org/jobs/technicalPM)~. Please do share with your network. 

![Technical PM](https://github.com/Green-Software-Foundation/discussions/assets/20337337/a8b89c53-30ce-4a64-b3a6-de964d775ea0)

**Comment by NAMRATA-WOKE** [comment]
We've got some amazing candidates for these positions. The Foundation is still accepting applications so feel free to apply or share the postings with your network >> https://www.linkedin.com/company/green-software-foundation/jobs/

**Comment by NAMRATA-WOKE** [comment]
The Content PM role has been filled. We're still seeking a Technical PM. If you know anyone who might be interested, please share the opportunity with them!
